packages/grafana-ui/src/components/Logs/getLogRowStyles.ts

```diff
@@ -35,18 +35,18 @@ export const getLogRowStyles = stylesFactory((theme: GrafanaTheme, logLevel?: Lo
       label: logs-row__match-highlight;
       background: inherit;
       padding: inherit;
-      color: ${theme.palette.yellow};
-      background-color: rgba(${theme.palette.yellow}, 0.1);
+      color: ${theme.colors.warning.main};
+      background-color: rgba(${theme.colors.warning.main}, 0.1);
```

Comment:
This seems to be the problem that causes the difference in highlighting though i can't find any colour that matches `theme.palette.yellow`. The closest one is `theme.colors.warning.text`

Reply:
Switching between dark and light mode also gives different colors for highlighting. @ifrost do you know if there are different styles that are being applied for each theme? I can't find anything here. From reading the code it feels like the colours should be the same in both dark and light theme

Reply:
> This seems to be the problem that causes the difference in highlighting though i can't find any colour that matches theme.palette.yellow. The closest one is theme.colors.warning.text

Might be worth checking with the UX if we could add a new color for highlights 🤔